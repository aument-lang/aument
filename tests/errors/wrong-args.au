func f(a,b,c) {
}
f(10)