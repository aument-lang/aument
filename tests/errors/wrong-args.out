parser error(4) in -: wrong number of arguments (expected 3, got 1)
3 | f(10)
    ^
